.github/workflows/bench.yml

```diff
@@ -0,0 +1,100 @@
+name: "Run Benchmark"
+
+on:
+  workflow_dispatch:
+    inputs:
+      bench-file:
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      bench-file:
+        required: true
+        type: string
+
+jobs:
+  benchmark:
+    runs-on: ubuntu-latest
+    # runs-on: self-hosted
+
+    strategy:
+      matrix:
+        node-version: [16.0.0, 16.18.1, 16.19.0, 16.20.0, 18.0.0, 18.14.2, 18.15.0, 18.16.0, 19.0.0, 19.8.0, 19.8.1, 19.9.0, 20.0.0]
+      max-parallel: 4
```

Comment:
Since the machine will only take one job at time, this value doesn't matter.

But I will set this value as `1`, just to ensure to run one benchmark at time.